airflow/configuration.py

```diff
@@ -1732,6 +1787,31 @@ def create_default_config_parser(configuration_description: dict[str, dict[str,
     return parser
 
 
+def write_default_airflow_configuration_if_needed():
+    if not os.path.isfile(AIRFLOW_CONFIG):
+        import rich
+
+        rich.print("Creating new Airflow config file in: %s", AIRFLOW_CONFIG)
```

Comment:
from rich import print here?

Reply:
Yes. To make it nicely coloured..

Reply:
No, i meant can we import it the way I mentioned?